core/test/src/test_cpu.cpp

```diff
@@ -658,23 +660,23 @@ TEST_F(CpuTest, unsupported_instruction) {
 }
 
 TEST_F(CpuTest, brk) {
+    registers.pc = 0x1234;
     stage_instruction(BRK);
     expected.pc = 0xDEAD;
 
-    const uint8_t expected_pc_stack_addr = expected.sp - 1;
-    const uint8_t expected_p_stack_addr = expected_pc_stack_addr - 1;
-
     expected.sp -= 2 + 1; // 1 word and 1 byte
 
-    ON_CALL(mmu, read_word(kBrkAddress)).WillByDefault(Return(0xDEAD));
+    // Dummy read
```

Comment:
Should we have an InSequence for everything here?

Reply:
Yeah. We'll deal with it in a separate PR. LGTM!